app/lib/helpers/helpers.ts

```
@@ -0,0 +1,288 @@
+function extractCID(ipfsUri: string) {
+  return ipfsUri.split(/\/\//g)[1];
+}
+
+export async function parseMetadata2(arr: NFT[]) {
+  const ast: NFT[] = [];
+  const delimiter = 'ipfs/';
+
+  for (let i = 0; i < arr.length; i++) {
+    let cid = arr[i].metadata.uri.split(delimiter)[1].substring(0, 59);
+
+    if (cid != undefined) {
```

Biome (1.9.4)

[error] Use !== instead of !=. != is only allowed when comparing against `null`

!= is only allowed when comparing against null

Using != may be unsafe if you are relying on type coercion
Unsafe fix: Use !==


(lint/suspicious/noDoubleEquals)